### DISCUSSION TITLE
aishell模型训练需要花多长时间？

### DISCUSSION THREAD

**Discussion by dabaoyasu** [PaddlePaddle/PaddleSpeech#1029] [Q&A]
32核CPU、64G内存的服务器，对 examples/aishell/s0/ 进行训练，花了几天时间了，还一直在训练中，怎么估算还需要训练多久才能完成？
![image](https://user-images.githubusercontent.com/94971279/143241500-417f2673-7011-4f9e-a7fb-5a0f31b702b9.png)

**Comment by dabaoyasu** [comment]
step 达到多少就会完成训练？

**Comment by dabaoyasu** [comment]
请大神们指点迷津，谢谢！

**Reply by zh794390558** [reply]
语音的模型得用 4 gpu 来训练，4 v100-32G 估计3天左右。纯cpu的机器无法用来训模型。

**Reply by dabaoyasu** [reply]
非常感谢。
训练的 epoch 一般设置多少合适，看默认值好像是80。
